assets/src/dashboard/components/scrollToTop/test/scrollToTop.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import ScrollToTop from '..';
+import theme from '../../../theme';
+
+const wrapper = (children) => {
+  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider>);
+};
```

Comment:
Are we doing this in other places a lot? may be a nice testing util `withTheme(children)`. If this is more of a one off tho ignore

Reply:
yeah this should be a testing util - i think it's almost everywhere we're testing in the components directory because of the theme needed for rendering styles.

Reply:
I'm going to make this util update as a separate PR this afternoon.